api/api_subsystems.go

```diff
@@ -0,0 +1,63 @@
+package api
```

Comment:
Should not prefix this file with `api` because it doesn't actually define an API.